tests/hierarchy_driver.cc

```diff
@@ -269,6 +269,7 @@ int main(int argc, char *argv[])
   int const fe_degree = params->get<unsigned int>("laplace.fe_degree", 4);
   params->put("fast_ap", true);
   params->put("eigensolver.type", "anasazi");
+  params->put("eigensolver.tolerance", 1e-2);
```

Comment:
What do we want here finally? `1.e-3`?

Reply:
Yeah I think that makes sense to change it.